core/src/main/java/org/lionweb/lioncore/java/utils/ValidationResult.java

```diff
@@ -0,0 +1,28 @@
+package org.lionweb.lioncore.java.utils;
```

Comment:
I think we envisioned that validation results/issues/etc. would also be a model, so we should define an M2 for this. That also means we can just reference the offending node through `id`.

Reply:
Right, but I think we did not yet define rules for that, so I would refactor this code once we get the validation m2